docs/contributor/04-50-cluster-vs-namespaced-scoped.md

```diff
@@ -0,0 +1,65 @@
+# Pros and cons of cluster scoped and namespaced scoped module custom resource
+
+
+<table style="table-layout: fixed; width: 100%">
+    <tbody>
+        <tr>
+            <td colspan="2" align="center">Cluster scoped Custom Resource</td>
+            <td colspan="2" align="center">Namespaced scoped Custom Resource</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;" align="center">Pros</td>
+            <td style="width: 25%;" align="center">Cons</td>
+            <td style="width: 25%;" align="center">Pros</td>
+            <td style="width: 25%;" align="center">Cons</td>
+        </tr>
+        <tr>
+            <td>[UX] Resource is more visible to customer, and the module is clearly visible as installed.<br></br> This might also guarantee better visibility that this resource manages resources across the cluster, not only in a single namespace.</td>
```

Comment:
I'd remove my comment, because this is very opinionated. Also since this argument is heavily bound to how the UI is implemented I'd not list it as the first one. I'd have a very strong argument as the first argument in the list.
```suggestion
            <td>[UX] The module and the resource is clearly visible as installed to customer in the current UI implementation. This better visibility might also </td>
```